src/js/index.js

```diff
@@ -0,0 +1,20 @@
+const backToTop = document.querySelector(".backToTop");
+
+function toggleBackToTopButton() {
+    if (window.scrollY >= 250) {
+        backToTop.classList.add("active");
+    } else {
+        backToTop.classList.remove("active");
+    }
+}
+
+
+window.addEventListener("scroll", toggleBackToTopButton);
+
+
+backToTop.addEventListener("click", () => {
+    window.scrollTo({
+        top: 0,
+        behavior: "smooth",
+    });
+});

```

Comment:
See previous note. But nice work!

Reply:
Question: if you move this entire script and move it to BaseFooter.astro component it would be better right?
My point: trying to unify the logic with the component itself without creating additional JS file (helps in code navigation since there's less files to traverse)

Reply:
it's probably better, I'm not really that well versed in where things go in astro. I personally dont mind where anything is, as long as it works, but I'm glad someone can always contribute with better insight. :)

Reply:
it does work like this, needs a look from Elian cuz Typescript is out of my hands
```js
const backToTop: HTMLButtonElement | any = document.querySelector(".backToTop");
... rest of the code
```
What i see is:
- Your button element tag is not closed  (causes whole footer merged inside that small circle)
- The backtoTop Arrow image is not at all visible (watching at 1920X1080) right now

Reply:
Hey peeps! Really appreciate the work done here! I was out for a couple of days. Will take a look and improve the PR further this week. 
I do have opinions, like moving the whole script into a new component together with the button. But also adding a prop to hide the button from showing up. 
Will discuss further soon!